src/components/pages/ExampleDataViz/CityInfo.js

```diff
@@ -1,15 +1,21 @@
 import React from 'react';
 import 'antd/dist/antd.css';
-import './index.css';
+// import './index.css';
 import { Card } from 'antd';
+import { HeartOutlined } from '@ant-design/icons';
 
 function CityInfo(props) {
   return (
     <div className="site-card-border-less-wrapper">
-      <Card title="City, State" bordered={false} style={{ width: 300 }}>
+      <Card
+        title={props.city + ', ' + props.state}
+        bordered={false}
+        style={{ width: 300 }}
+      >
         <p>Average Rent Price:</p>
         <p>Overall Crime Rate:</p>
         <p>Walkability Score:</p>
+        <HeartOutlined />
```

Comment:
the way you utilizing props and importing this component here is a good practice.